client/controller/config/sample_oracle_config.json

```diff
@@ -0,0 +1,9 @@
+{
+  "database_type" : "oracle",
+  "database_url" : "jdbc:oracle:thin:@localhost:1521:orcldb",
+  "username" : "sys as sysdba",
```

Comment:
Since this is a sample configuration, use the same username/password/workload_name placeholders as sample_postgres_config.json